pages/form/returning-members.js

```diff
@@ -72,22 +77,15 @@ const SUBTEAMS = [
   'Team Hub',
```

Comment:
The `SUBTEAMS` here is different from that in `registration.js`

Reply:
That's probably because this list changes from year to year depending on what the directors decide for team organization/structure in a particular term. I pulled this list from different places in the Google Drive.

That's why I asked whether the admins are currently updating the list of subteams using the CMS-Server every term.

Reply:
Eventually, these options will not be hard-coded anymore on the frontend. Instead, they will be fetched from the database.

This form will eventually be editable (admins can edit each question's available option set, each question's description, add new questions, etc.), thus allowing admins to enter brand new subteam options or delete old subteam options if they want.

Reply:
That makes sense. I still think that the teams should be fetched from one source of truth though, so that we wouldn't have to update the subteams from both Teamhub and the cms-server whenever there are updates. I added some more detail in the other comment.